src/lib/filterTasksByClientClassification.spec.ts

```diff
@@ -0,0 +1,88 @@
+import filterClassifiedTasks from "./filterTasksByClientClassification";
+
+/* eslint-disable @typescript-eslint/camelcase */
+const clientFixture = {
+  country_code: "GB",
+  asn: 5089,
+  connection_type: "broadband",
+  device_type: "unkown"
+};
+
+/* eslint-disable @typescript-eslint/camelcase */
```

Comment:
redundant. Did you mean to put `/* eslint-enable @typescript-eslint/camelcase */` above to turn this back on?